src/converters/lintConfigs/rules/ruleConverters.ts

```diff
@@ -64,6 +64,17 @@ import { convertCurly } from "./ruleConverters/curly";
 import { convertCyclomaticComplexity } from "./ruleConverters/cyclomatic-complexity";
 import { convertDeprecation } from "./ruleConverters/deprecation";
 import { convertEofline } from "./ruleConverters/eofline";
+import { convertActionHygiene } from "./ruleConverters/eslint-plugin-ngrx/action-hygiene";
```

Comment:
I think the folder name would be `ngrx-tslint-rules` as, like `codelyzer`, it refers to the **tslint** package, but as there are several other folders referencing the name of **eslint** package, I was confused and I preferred to follow the majority 🤷‍♂️

Reply:
Yeah good point. Filed a followup to take another look at these: #1197

Reply:
Nice! Do you want me to update the folder name to `ngrx-tslint-rules` then?

Reply:
Sure, that'd be swell. Yes please!